GitDiffMargin/Git/HunkRange.cs

```diff
@@ -2,20 +2,19 @@ namespace GitDiffMargin.Git
 {
     public class HunkRange
     {
-        public HunkRange(string hunkRange, int contextLines)

```

Comment:
Yes in the future I wanted to introduce a possibility to define some value for it, which would be useful [here](https://github.com/laurentkempe/GitDiffMargin/blob/master/GitDiffMargin/Git/GitCommands.cs#L112)